app/components/article-card.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import {H3} from './title'
+
+export interface ArticleCardProps {
+  title: string
+  date: Date | number | string
+  readTime: string
+  imageUrl: string
+  articleUrl: string
+}
+
+function ArticleCard({
+  title,
+  date,
+  readTime,
+  imageUrl,
+  articleUrl,
+}: ArticleCardProps) {
+  const dateString = new Date(date).toLocaleDateString('en-US', {
+    dateStyle: 'long',
+  })
```

Comment:
Should we extract date formatting to an util, and maybe make it smarter?

Reply:
Added to a project board so it's not forgotten. Let's make this part of another PR where it's more relevant.

https://github.com/kentcdodds/remix-kentcdodds/projects/1#card-63944862